hack/generator/pkg/astmodel/optional_type.go

```diff
@@ -65,6 +65,14 @@ func (optional *OptionalType) AsType(codeGenerationContext *CodeGenerationContex
 	}
 }
 
+// AsZero renders an expression for the "zero" value of the type
+// by returning a literal "nil"
+func (optional *OptionalType) AsZero(types Types, ctx *CodeGenerationContext) dst.Expr {
```

Comment:
minor: Don't use `types` here so could be `_`

Reply:
Fixed. I suspect our linting will complain if I don't.